src/less/variables-payex.less

```diff
@@ -195,6 +195,7 @@ body {
 	--bg-grey: var(--brand-secondary-light-2, #c8c8c8);
 	--bg-white: #ffffff;
 	--bg-quaternary: var(--list-gray, #999999);
+	--bg-togglebox-theme-white: #d9d9d9;
```

Comment:
if this rule is only used for documentation and not meant to be used by projects consuming the design system then move it to the "documentation-payex.less" file